recipes/rilseq/meta.yaml

```diff
@@ -1,33 +1,35 @@
 {% set name = "rilseq" %}
-{% set version = "0.75" %}
+{% set version = "0.78" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
   url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/RILseq-{{ version }}.tar.gz"
-  sha256: 969a009f06c4cfcfeb59c5c9f0680e2196b77a55b663fb459f97c0e5cdd3e02a
+  sha256: 7386116c01b399e1c76f6affd8a75d151ab8e10bee70d2852baabffa35b3e8b1
 
 build:
-  number: 0
+  number: 1
```

Comment:
This should stay at 0, since you are increasing the version.

Reply:
There's already a 0.78 version that was automatically generated